sources/api/apiserver/src/server/exec/child.rs

```diff
@@ -241,6 +241,13 @@ impl ChildHandles {
     }
 }
 
+impl Drop for ChildHandles {
```

Comment:
Just to confirm, you still get the "ERROR" after the `apiclient exec` command completes, right?